rotkehlchen/db/utils.py

```diff
@@ -45,6 +47,11 @@
     from rotkehlchen.chain.bitcoin.xpub import XpubData
     from rotkehlchen.db.dbhandler import DBHandler
 
+
+logger = logging.getLogger(__name__)
```

Comment:
All changes in this file need to now be reverted. You forgot to do that when you moved it to the connection